communicator/ssh/communicator.go

```diff
@@ -96,6 +96,21 @@ func (c *Communicator) Connect(o terraform.UIOutput) (err error) {
 			c.connInfo.KeyFile != "",
 			c.connInfo.Agent,
 		))
+
+		if c.connInfo.BastionHost != "" {
+			o.Output(fmt.Sprintf(
+				"Using configured bastion host..."+
+					"  Host: %s\n"+
+					"  User: %s\n"+
+					"  Password: %t\n"+
```

Comment:
I thought Terraform user output (w/out `TF_LOG=1`) was always treated as output without secrets (based on the fact that Terraform doesn't print out used AWS credentials, even though it would be quite helpful for debugging sometimes), but maybe I'm just wrong and it's ok to print passwords out? I can just see that being a potential issue during public demos :smiley: and generally when storing these logs anywhere in unsecured log storage.